src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Complete/PayPalExpressCompleteResponseSaver.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Payment\Handler\Saver\Complete;
+
+use Generated\Shared\Transfer\ComputopApiPayPalExpressCompleteResponseTransfer;
+use Generated\Shared\Transfer\ComputopPayPalExpressInitResponseTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Orm\Zed\Computop\Persistence\SpyPaymentComputop;
+use Spryker\Zed\Kernel\Persistence\EntityManager\TransactionTrait;
+use SprykerEco\Zed\Computop\ComputopConfig;
+use SprykerEco\Zed\Computop\Dependency\Facade\ComputopToOmsFacadeInterface;
+use SprykerEco\Zed\Computop\Persistence\ComputopQueryContainerInterface;
+
+class PayPalExpressCompleteResponseSaver implements CompleteResponseSaverInterface
+{
+    use TransactionTrait;
+
+    /**
+     * @var \SprykerEco\Zed\Computop\Persistence\ComputopQueryContainerInterface $queryContainer
+     */
+    protected $queryContainer;
```

Comment:
I'm not sure we can use query container for newly introduced models
Most probably you need to introduce repository method with needed functionality and inject it here